### PR TITLE
[JEP-227] Update some status and add some PRs

### PR DESCRIPTION
**Pull request by Wadeck**
@jglick

### CODE REVIEW

File: jep/227/compatibility.adoc [hunk 1]
```diff
@@ -121,8 +121,8 @@ Cleanups integrated as of link:https://github.com/jenkinsci/credentials-plugin/r
 could be switched to `hasPermission` for full compatibility.
 
 |link:https://plugins.jenkins.io/extensible-choice-parameter/[extensible-choice-parameter]
-|Probably compatible
-|Unclear what the purpose of link:https://github.com/jenkinsci/extensible-choice-parameter-plugin/blob/3fda75de34d4099d078a038fc3eeb3c38679c1f8/src/test/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/SystemGroovyChoiceListProviderJenkinsTest.java#L217-L219[this test code] is.
+|Compatible
+|Only tests require adjustment, e.g. link:https://github.com/jenkinsci/extensible-choice-parameter-plugin/blob/3fda75de34d4099d078a038fc3eeb3c38679c1f8/src/test/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/SystemGroovyChoiceListProviderJenkinsTest.java#L217-L219[this test], pending link:https://github.com/jenkinsci/extensible-choice-parameter-plugin/pull/42[extensible-choice-parameter-plugin #4] for tests without Acegi.
```
Comment (jglick) [inline jep/227/compatibility.adoc:125]:
```suggestion
|link:https://github.com/jenkinsci/extensible-choice-parameter-plugin/pull/42[extensible-choice-parameter-plugin #4] required for tests to pass.
```

(I suppose? Unclear to me.)

File: jep/227/compatibility.adoc [hunk 2]
```diff
@@ -150,7 +150,7 @@ would need to be adjusted to make tests pass.
 |link:https://plugins.jenkins.io/jobConfigHistory/[jobConfigHistory]
 |Probably compatible
 |link:https://github.com/jenkinsci/jobConfigHistory-plugin/blob/d7cef005f875b00a04e6bd7a81eee69510b65c59/src/test/java/hudson/plugins/jobConfigHistory/JobConfigHistoryProjectActionTest.java#L317[This code]
-would need to be adjusted to make tests pass.
+would need to be adjusted to make tests pass. Pending link:https://github.com/jenkinsci/jobConfigHistory-plugin/pull/127[jobConfigHistory-plugin #127] for test passing and ensuring full compatibility when users are lacking permissions.
```
Comment (jglick) [inline jep/227/compatibility.adoc:153]: Suffices to link to the PR, no need to retain code links now.

File: jep/227/compatibility.adoc [hunk 3]
```diff
@@ -236,10 +236,10 @@ should be adjusted.
 |As of link:https://github.com/jenkinsci/plugin-util-api-plugin/releases/tag/plugin-util-api-1.2.5[1.2.5].
 
 |link:https://plugins.jenkins.io/purge-job-history/[purge-job-history]
-|Mostly compatible
+|Compatible
 |link:https://github.com/jenkinsci/purge-job-history-plugin/blob/e694f76d73b7b6402741d5ba88df2f702dab993e/src/main/java/jenkins/plugins/purgejobhistory/PurgeJobHistory.java#L124-L130[This code] and
 link:https://github.com/jenkinsci/purge-job-history-plugin/blob/e694f76d73b7b6402741d5ba88df2f702dab993e/src/main/java/jenkins/plugins/purgejobhistory/PurgeJobHistory.java#L193-L199[this code]
-should be switched to `hasPermission` for full compatibility.
+should be switched to `hasPermission`. Pending link:https://github.com/jenkinsci/purge-job-history-plugin/pull/10[purge-job-history-plugin #10] for full compatibility + bug correction. No functional change.
```
Comment (jglick) [inline jep/227/compatibility.adoc:242]: Again, can keep the third column simple here: just link to the PR.

File: jep/227/compatibility.adoc [hunk 4]
```diff
@@ -236,10 +236,10 @@ should be adjusted.
 |As of link:https://github.com/jenkinsci/plugin-util-api-plugin/releases/tag/plugin-util-api-1.2.5[1.2.5].
 
 |link:https://plugins.jenkins.io/purge-job-history/[purge-job-history]
-|Mostly compatible
+|Compatible
```
Comment (jglick) [inline jep/227/compatibility.adoc:239]: No, mostly compatible _as of now_. (Compatible if and when that PR is released.)

**Review by jglick — APPROVED** [review]
Thanks!

File: jep/227/compatibility.adoc (hunk 1 above)
Comment (jglick) [reply jep/227/compatibility.adoc:125]: Acc. to https://github.com/jenkinsci/extensible-choice-parameter-plugin/pull/42#issuecomment-698345257 that PR is _not_ even required for tests to pass; it is just cleanup. So should be reworded.

File: jep/227/compatibility.adoc (hunk 4 above)
Comment (Wadeck) [reply jep/227/compatibility.adoc:239]: In this plugin, now you get a 500 if you are lacking permission and with the core update, you will get 403.
The PR corrects the bug that makes the "feature" working.

So if you consider such corner case in the "mostly", fine :) Just wanted to give details ;)

**Review by jglick — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**